### PR TITLE
Added product details page and other features

### PR DESCRIPTION
**Pull request by teenox**
Recording 2024-05-09 173255

Implemented the following:
- Shopping cart
- Product details page
- Checkout page(WIP)

### CODE REVIEW

File: src/Components/Product/Productdetail.jsx
Comment (gavintendekai) [inline src/Components/Product/Productdetail.jsx:1]: nice

File: src/pages/checkout/checkout.jsx
Comment (gavintendekai) [inline src/pages/checkout/checkout.jsx:1]: will it be possible to add orders to the database? after a payment goes through, the order gets put into the db

**Review by gavintendekai — COMMENTED** [review]
looks good

File: src/pages/checkout/checkout.jsx
Comment (teenox) [reply src/pages/checkout/checkout.jsx:1]: Yes it is possible. We just need an API to do that.

**Pull request merged**